### ISSUE TITLE
Difficulty installing on M1 mac (solved)

### ISSUE THREAD

**Issue by johnmcdonnell** [google/sentencepiece#763]
Hi all just wanted to share this in case anyone else runs into it. I was having trouble installing sentencepiece on my M1 mac. Tried `pip install sentencepiece`, tried downloading, tried installing from source. Kept running into:
```  clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -I/Users/jvm/venv/include -I/opt/homebrew/opt/python@3.9/Frameworks/Python.framework/Versions/3.9/include/python3.9 -c src/sentencepiece/sentencepiece_wrap.cxx -o build/temp.macosx-12-arm64-cpython-39/src/sentencepiece/sentencepiece_wrap.o -std=c++11 -mmacosx-version-min=10.9
  src/sentencepiece/sentencepiece_wrap.cxx:2809:10: fatal error: 'sentencepiece_processor.h' file not found
  #include <sentencepiece_processor.h>
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~
  1 error generated.
  error: command '/usr/bin/clang' failed with exit code 1
  error: subprocess-exited-with-error
```

Tried installing the c library from source, when I tried running `sudo update_dyld_shared_cache` I got a note that it was deprecated, and then couldn't seem to find the header file.

In the end all I needed to do to solve it was `brew install sentencepiece`. So might be ideal to add that to the documentation.

**Comment by taku910** [comment]
The wheel binary package for Mac arm processors is available from v0.1.97. 
Let me close this bug. If you see the problems, please recreate another one. Thank you.

**Comment by erlebach** [comment]
Running with version 0.1.99 on a Mac with M1 processor installed via pip, I get the error 'AttributeError: module 'sentencepiece' has no attribute 'encode'.  I may have a coding error. 
```
  spm.SentencePieceTrainer.train(input='botchan.txt', model_prefix='m',
                                vocab_size=500, model_type='bpe')
```
Thanks!
